src/components/view/Navigation/index.js

```diff
@@ -58,23 +44,40 @@ class Navigation extends React.Component {
         this.props.history.push(tabEl.props['data-route']);
     };
 
+    getMenuItemHandler (routeName)
+    {
+        var that = this;
+        return function ()
+        { 
+            that.setState({open: false});
+            that.props.history.push(routeName);
+        };    
+    }
+
+    handleLogout = () => {this.setState({open: false}); this.props.logout()}
```

Comment:
Use function definition syntax as `navigateToRoute` instead of prop definition syntax. Use line breaks between statements when the function definition body has more than one statement.